ci/test/00_setup_env_native_previous_releases.sh

```diff
@@ -16,5 +16,6 @@ export RUN_UNIT_TESTS_SEQUENTIAL="true"
 export RUN_UNIT_TESTS="false"
 export GOAL="install"
 export DOWNLOAD_PREVIOUS_RELEASES="true"
-export BITCOIN_CONFIG="--enable-zmq --with-gui=qt5 --enable-reduce-exports --enable-debug \
-CFLAGS=\"-g0 -O2 -funsigned-char\" CPPFLAGS='-DBOOST_MULTI_INDEX_ENABLE_SAFE_MODE' CXXFLAGS=\"-g0 -O2 -funsigned-char\""
+export BITCOIN_CONFIG="-DWITH_ZMQ=ON -DBUILD_GUI=ON -DREDUCE_EXPORTS=ON -DCMAKE_BUILD_TYPE=Debug \
+-DCMAKE_C_FLAGS='-funsigned-char' -DCMAKE_C_FLAGS_DEBUG='-g0 -O2' \
```

Comment:
Why is this setting both the generic and build-type specific flags?

Reply:
There were related discussions [here](https://github.com/hebasto/bitcoin/pull/142#discussion_r1554395773) and [here](https://github.com/hebasto/bitcoin/pull/142#discussion_r1605534766).